src/schoology-extractor/schoology_extractor/mapping/sections.py

```diff
@@ -0,0 +1,65 @@
+# SPDX-License-Identifier: Apache-2.0
+# Licensed to the Ed-Fi Alliance under one or more agreements.
+# The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
+# See the LICENSE and NOTICES files in the project root for more information.
+
+import pandas as pd
+
+from . import constants
+
+
+def map_to_udm(sections_df: pd.DataFrame) -> pd.DataFrame:
+    """
+    Maps a DataFrame containing Schoology sections into the Ed-Fi LMS Unified Data
+    Model (UDM) format.
+
+    Parameters
+    ----------
+    sections_df: DataFrame
+        Pandas DataFrame containing Schoology sections
+
+    Returns
+    -------
+    DataFrame
+        A LMSUsers-formatted DataFrame
+
+    Notes
+    -----
+    DataFrame columns are:
+        SourceSystemIdentifier: A unique number or alphanumeric code assigned to a user by the source system
+        SourceSystem: The system code or name providing the user data
+        SISSectionIdentifier: Section identifier as recorded in the student information system (SIS)
+        Title: Section title
+        SectionDescription: Section long description
+        Term: Calendar / grading period term
+        LMSSectionStatus: Status of the section in the learning management system (LMS)
+        EntityStatus: The status of the record
+        CreateDate: datetime at which the record was first retrieved
+        LastModifieDate: datetime when the record was modified, or when first retrieved
+    """
+
+    df = sections_df[["id", "section_title", "description", "active"]].copy()
+
+    df["SourceSystem"] = constants.SOURCE_SYSTEM
+    df["EntityStatus"] = constants.ACTIVE
+
+    df.rename(
+        columns={
+            "id": "SourceSystemIdentifier",
+            "section_title": "Title",
+            "description": "SectionDescription",

```

Comment:
On lines 43 and 44 I used constants to represents _values_, but elsewhere I have used plain strings to represent _column names_. This just felt natural as I was writing it. Are you guys comfortable with that, or do you find it to be weirdly inconsistent?

@bradbanister  @lalrg

Reply:
👍  I find it natural as well

Reply:
Looks good to me

Reply:
OK, thanks for confirming.